react-intervaler/package.json

```diff
@@ -1,21 +1,29 @@
 {
   "name": "react-intervaler",
+  "homepage": "http://dariuscox.github.io/intervaler",
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "bootstrap": "^4.5.3",
```

Comment:
devDependencies vs dependencies

https://medium.com/@dylanavery720/npmmmm-1-dev-dependencies-dependencies-8931c2583b0c#:~:text=The%20difference%20between%20these%20two,an%20npm%20install%20%2D%2Dsave.